app/views/short_interactions/_form.html.haml

```diff
@@ -0,0 +1,45 @@
+-# Copyright © 2011-2016 MUSC Foundation for Research Development~
```

Comment:
Is this form partial being used? I can't seem to find where it would be rendered.

Reply:
we had this for so long, it might be the original one that no longer needed. I will take a closer look